app/components/Modals/SendModal/ReadCode/ReadCode.jsx

```diff
@@ -0,0 +1,81 @@
+// @flow
+import React from 'react'
+import classNames from 'classnames'
+
+import Button from '../../../Button'
+import QrCodeScanner from '../../../QrCodeScanner'
+import GridIcon from '../../../../assets/icons/grid.svg'
+import CozDonationQrCode from '../../../../assets/images/coz-donation-qr-code.png'
```

Comment:
for a future PR it would be nice to alias the path for assets so we don't have to find the exact path. 'assets/images...'. No need to change anything. Just some food for thought

Reply:
I've added an alias for `assets` to clean-up these admittedly ugly imports.